<hr>

# Module 76.1 Study - Put-Call Parity

<hr>

In [6]:
import numpy as np

### LOS 76.a: Explain put–call parity for European options.

Our derivation of **put-call parity** for European options is based on the payoffs of two portfolio combinations: a fiduciary call and a protective put.

A *fiduciary call* is a combination of a call with exercise price $X$ and a pure-discount, riskless bond that pays $X$ at maturity (option expiration). The payoff for a fiduciary call at expiration is $X$ when the call is out of the money, and $X + (S − X) = S$ when the call is in the money.

A *protective put* is a share of stock together with a put option on the stock. The expiration date payoff for a protective put is $(X – S) + S = X$ when the put is in the money, and $S$ when the put is out of the money.

>**Professor's Note**
/
When working with put-call parity, it is important to note that the exercise prices on the put and the call and the face value of the riskless bond are all equal to $X$.

If at expiration $S$ is greater than or equal to $X$:

*  protective put pays $S$ on the stock while the put expires worthless, so the payoff is $S$.
* The fiduciary call pays $X$ on the bond portion while the call pays $(S − X)$, so the payoff is $X + (S − X) = S$.

If at expiration $X$ is greater than $S$:

* The protective put pays $S$ on the stock while the put pays $(X − S)$, so the payoff is $S + (X − S) = X$.
* The fiduciary call pays $X$ on the bond portion while the call expires worthless, so the payoff is $X$.

In either case, the payoff on a protective put is the same as the payoff on a fiduciary call. Our no-arbitrage condition holds that portfolios with identical payoffs regardless of future conditions must sell for the same price to prevent arbitrage. We can express the put-call parity relationship as:

$\qquad c + X(1 + R_f)^{–T} = S + p$

Equivalencies for each of the individual securities in the put-call parity relationship can be expressed as:

$\qquad S = c - p + X(1 + R_f)^{–T}$
\
\
$\qquad p = c - S + X(1 + R_f)^{–T}$
\
\
$\qquad c = S + p - X(1 + R_f)^{–T}$
\
\
$\qquad S = c - p + X(1 + R_f)^{–T}$
\
\
$\qquad X(1 + R_f)^{–T} = S + p - c$


Note that the options must be European style and the puts and calls must have the same exercise price and time to expiration for these relations to hold.

The single securities on the left-hand side of the equations all have exactly the same payoffs as the portfolios on the right-hand side. The portfolios on the right-hand side are the **synthetic** equivalents of the securities on the left. For example, to synthetically produce the payoff for a long position in a share of stock, use the following relationship:

$S = c − p + X(1 + R_f)^{–T}$

This means that the payoff on a long stock can be synthetically created with a long call, a short put, and a long position in a risk-free discount bond.

The other securities in the put-call parity relationship can be constructed in a similar manner.

>**Professor's Note**
\
After expressing the put-call parity relationship in terms of the security you want to synthetically create, the sign on the individual securities will indicate whether you need a long position (+ sign) or a short position (– sign) in the respective securitie

#### Example: Call option valuation using put-call parity

Suppose that the current stock price is €52 and the risk-free rate is 5%. You have found a quote for a 3-month put option with an exercise price of €50. The put price is €1.50, but due to light trading in the call options, there was not a listed quote for the 3-month, €50 call. Estimate the price of the 3-month call option.

In [13]:
# Rearranging put-call parity, we find that the call price is:
# call = put + stock−present value (X)

call = 1.50 + 52 - (50/(1.05**0.25))
print("The value of the call option is: €",round(call,2))


# Calculate the discount factor
df = np.reciprocal(1.05**0.25)
# print the discount factor of PV
print(df)

# Now print the present PV of the strike price *(our ZCB)
PV = 50 * df
print(PV)


# We can try this another way mathematically to get PV of the strike price.
PV2 = (50/(1.05**0.25))
print(PV2)

The value of the call option is: € 4.11
0.9878765474230741
49.393827371153705
49.3938273711537


**Answer:**

Rearranging put-call parity, we find that the call price is:

$$
\text{call = put + stock−present value (X)}
$$
\
\
$$
\text{call} = €1.50 + €52 − \frac{€50}{1.05^{0.25}} = €4.11
$$
\
\
This means that if a 3-month, €50 call is available, it should be priced at (within transaction costs of) €4.11 per share.
\
\
€49.39 is the discounted 3 month $PV$ of the strike price *(our ZCB)*

### LOS 76.b: Explain put–call forward parity for European options.

**Put.call-forward parity** is derived with a forward contract rather than the underlying asset itself. Consider a forward contract on an asset at time $T$ with a contract price of $F_0(T)$. At contract initiation the forward contract has zero value. At time $T$, when the forward contract settles, the long must purchase the asset for $F_0(T)$. The purchase (at time $= 0$) of a pure discount bond that will pay $F_0(T)$ at maturity (time = $T$) will cost $F_0(T)(1 + R_f)^{–T}$.

By purchasing such a pure discount bond and simultaneously taking a long position in the forward contract, an investor has created a synthetic asset. At time = $T$ the proceeds of the bond are just sufficient to purchase the asset as required by the long forward position. Because there is no cost to enter into the forward contract, the total cost of the synthetic asset is the present value of the forward price, $F_0(T)(1 + R_f)^{–T}$.

The put-call-forward parity relationship is derived by substituting the synthetic asset for the underlying asset in the put-call parity relationship. Substituting $F_0(T)(1 + R_f)^{–T}$ for the asset price $S_0$ in $S + p = c + X(1 + R_f)^{–T}$ gives us:
\
\
$$
F_0(T)(1 + R_f)^{–T} + p_0 = c_0 + X(1 + R_f)^{–T}
$$
\
which is put-call-forward parity at time $0$, the initiation of the forward contract, based on the principle of no arbitrage. By rearranging the terms, put-call-forward parity can also be expressed as:

$$
p_0 − c_0 = [X −F_0(T)](1 + R_f)^{–T}
$$

#### Application of Options Theory to Corporate Finance

We can view the claims of a firm's equity holders and debt holders as a call option and a put option, respectively. Consider a firm that has a value of $V_t$ at time = $t$ and has issued debt in the form of a <u>zero-coupon bond</u> that will pay $D$ at time = $T$. At time = $T$, if $V_T > D$ the equity holders receive $V_T – D$ and if $V_T < D$, the firm is <u>insolvent</u> and equity holders receive nothing. 

The payoff to the equity holders at time = $T$ can be written as $\text{Max}(0, V_T – D)$ which is equivalent to a call option with the firm value as the underlying and an exercise price of $D$.

At time = $T$, if $V_T > D$ the debt holders receive $D$ and if $V_T < D$, the firm is insolvent and debt holders receive $V_T$. The payoff to the debt holders at time = $T$ can be written as $\text{Min}(V_T, D)$. This is equivalent to a portfolio that is long a risk-free bond that pays $D$ at $t = T$, and short (has sold) a put option on the value of the firm, $V_T$, with an exercise price of $D$. If $V_T > D$ the portfolio pays $D$ and the put expires worthless, and if $V_T< D$ the portfolio pays $D – (D – V_T) = V_T$ and the debtholders effectively pay $D – V_T$ on the short put position.



<IMG SRC ="https://github.com/PachaTech/CFA-Level-1/blob/main/Derivatives/Module%2076/pics/1.png?raw=true">
<IMG SRC ="https://github.com/PachaTech/CFA-Level-1/blob/main/Derivatives/Module%2076/pics/2.png?raw=true">

<hr>
    
# Module 76.1: Put-Call Parity Quiz & Qbank

<hr>

**Question 1**\
The put-call parity relationship for European options must hold because a protective put will have the same payoff as a(n):

**Answer**\
fiduciary call.

**Explanation**\
Given call and put options on the same underlying asset with the same exercise price and expiration date, a protective put (underlying asset plus a put option) will have the same payoff as a fiduciary call (call option plus a risk-free bond that will pay the exercise price on the expiration date) regardless of the underlying asset price on the expiration date.
\
(Module 76.1, LOS 76.a)

<hr>

**Question 2**\
The put-call-forward parity relationship *least likely* includes:


**Answer**\
the underlying asset.

**Explanation**\
The put-call-forward parity relationship is $F_0(T)(1 + R_f)^{–T} + p_0 = c_0 + X(1 + R_f)^{–T}$, where $X(1 + R_f)^{–T}$ is a risk-free bond that pays the exercise price on the expiration date, and $F_0(T)$ is the forward price of the underlying asset. 
\
(Module 76.1, LOS 76.b)

<hr>

**Question 3**\
Using put-call parity, it can be shown that a synthetic European call can be created by a portfolio that is:


**Answer**\
long the stock, long the put, and short a pure discount bond that pays the exercise price at option expiration.

**Explanation**\
A stock and a put combined with borrowing the present value of the exercise price will replicate the payoffs on a call at option expiration.
\
(Module 76.1, LOS 76.a)


<hr>

**Question 4**\
An investor calculates that the premium of a European put option is less than its value based on put-call parity. In exploiting this arbitrage opportunity, the investor is *most likely* to:

**Answer**\
sell the call option.

**Explanation**\
Put-call parity indicates that $P = C + PV(X) – S$. With  $P < [C + PV(X) – S]$, the arbitrage transaction is to buy the put and sell the call, borrow the $PV$ of the exercise price ($X$), and buy the stock.
\
(Module 76.1, LOS 76.a)



<hr>

**Question 5**\
A synthetic European put option includes a short position in:

**Answer**\
the underlying asset.

**Explanation**\
A synthetic European put option consists of a long position in a European call option, a long position in a risk-free bond that pays the exercise price on the expiration date, and a short position in the underlying asset.
\
(Module 76.1, LOS 76.a)

<hr>

**Question 6**\
Which of the following instruments is a component of the put-call-forward parity relationship?

**Answer**\
The present value of the forward price of the underlying asset.

**Explanation**\
The put-call-forward parity relationship is: $\Large\frac{F_0(T)}{(1 + R_f)^{T} + p} = \frac{c + X}{(1 + R_f)^{T}}$, where $F_0(T)$ is the **forward price** of the underlying asset.
\
(Module 76.1, LOS 76.a)
<hr>